recipe/migrations/python312.yaml

```diff
@@ -0,0 +1,42 @@
+migrator_ts: 1695046563
+__migrator:
+    migration_number: 1
+    operation: key_add
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.11.* *_cpython
+            - 3.12.* *_cpython  # new entry
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: false
+    longterm: True
+    pr_limit: 20
+    max_solver_attempts: 3  # this will make the bot retry "not solvable" stuff 12 times
+    exclude:
+      # this shouldn't attempt to modify the python feedstocks
+      - python
+      - pypy3.6
+      - pypy-meta
+      - cross-python
+      - python_abi
+    exclude_pinned_pkgs: false
+    additional_zip_keys:
+      - channel_sources
```

Comment:
See https://github.com/conda-forge/conda-smithy/pull/1764#issuecomment-1726250148

Reply:
I would release conda-smithy 3.26.2 later and would wait for a merge of https://github.com/conda-forge/cross-python-feedstock/pull/74. Are then all concerns regarding the migration addressed?

Reply:
Yes